.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: node_js
+node_js:
+- node
+script:
+- npm run test
+deploy:
+  provider: npm
+  email: 'systems@azavea.com'
+  api_key: ''
+  env:
+    global:
+      secure: qkinmGg7embShBpjy9P4617XlcvpmgThBiO5NfGVoX5ciPshhuAc1je0wMyI4SiPElibq/8AS/L6Qa8pyPG/RYeo4zg4ZRFoMvpffKlmvBhx1Y+kEpfL3GBLKNb7MS6nHq/1YlqtJ0JU/REVjgn1wMdLWLaNEoeP0VoGp6CbGl+IsdqxcepPDQ1cZ4mHnlCDhkbRqpyEyTjV/Z77/ZXpHrWvb1WkInuYP7BxX1t0vTGsTZpW9kGLHU+h6XRoXTJsbZLlZUiVtsWGJypjo8RcagH0bIi8btiGQdTKjboHDjHrt04eciF7LgsmEUll2Go/v6UxQRKHP+l4AmxQ+nhN1P8ZjwnXWMlUaQls86BkieYiM3lt4SZPQpctPGa5aDa93OSb7TFzYG82mFUrIbBAtmenw2aY6GOxEWs3/k3TiMR0KnYHRZ67sLVmN6My7wgCxcpVuWS3GVPbBXj8TbL+tgdiWAap/xj+jSUrGu9ECq645+8U36UeC2HK/mvJ17TAKgHmJ06m5qXvq4IziIAQCHicZxTANorgfq1Ok/mRPiLRAqz9inMK3E/oKihj48ibXxhE5ucS8mnMRHewVNeyJRbu+D/OKElA0BEBQdQWmanv3t2WTzOR/9qzeNhXw5Zpw6lr5j2a++JpbrtxKbrv9nOmBKjLIWSmWLPK2cBqlII=
```

Comment:
I'm not sure this is correct.  I believe we want he encrypted auth token in the `deploy > api_key` property, not as an environment variable.

Reply:
Travis placed the auth token there.

I generated it following the steps at: http://microapps.com/blog/auto-publishing-to-npm-by-travis-ci/. This could be the wrong workflow for our use case, though.

Reply:
I think you'd use 

`$ travis encrypt YOUR_AUTH_TOKEN --add deploy.api_key`

to get it to place it in the correct path 
(see: https://docs.travis-ci.com/user/deployment/npm/#NPM-auth-token)

Reply:
Got it. That resulted in this:

```
language: node_js
deploy:
  api_key:
    secure: QXcvN8IXOK95N5MPCXLWdQ3+PxuISMboekkryugpv8NhR9H5kxts6kTQ8pe3P8kc67S3DhD6hoD70AQPvdiiv1+HeOPgHgQn9GONHnmygvQZ/soPK5tsilKu657Qt3Otxx/qliPXZjFZjBVZwsYU04f0PJmE2kpwK72rr95Z6DBMQLBmOLB+CIo7CtoMXG7Jmbt0lsTkR+JoLZJNIKqA3RvlVFkcPFIHrLmBfz0YyalshNc1nYjleW3Y07J+D2BSRrqAWmpzdKc9fUNelBF0aHuIEJvRM7XLPpFrvAlSOVtbMAW8AkZe9rwkOg6MkWOA1POwRLXujwTNbrgfVQivyWqggB/CIycvz1JyTlG61X5iWQK+frcYNIEUcU66lTkjyFuCsuOBOuO6sfGTSUYTMBDk5YES2ZQVGUSkgL20CAMY8L7vuea6aimrsYgw48PQzCvQGFdNw3DVgidOJxbjuriT8okWWBm9ICSlGC2llIsvjvIGS+77bus+VfotPdgr5wV+RhfuAFlgpYkcQnDuxj0LA/xd9xHcw1xtav9XjmpMCPZE1ATxLzciG/yj2+npJ39NdtEhlbuj4pWMxI6IFLJ9m7ksK6QCe5W+yHvKKuJAMjKj2tza3v4hnhxgMed1B8jTR9lwSxmqOqMxfRGj0v2G/wgHxn/Qnqsx7k9X++U=
api_key:
email: systems@azavea.com
provider: npm
```